--- gi/_glib/Makefile.am.link	2011-08-22 05:35:57.000000000 +0200
+++ gi/_glib/Makefile.am	2011-08-22 05:36:11.000000000 +0200
@@ -17,7 +17,7 @@
 endif
 
 libpyglib_gi_2_0_@PYTHON_BASENAME@_la_CFLAGS = $(GLIB_CFLAGS)
-libpyglib_gi_2_0_@PYTHON_BASENAME@_la_LIBADD = $(GLIB_LIBS) $(FFI_LIBS)
+libpyglib_gi_2_0_@PYTHON_BASENAME@_la_LIBADD = $(GLIB_LIBS) $(FFI_LIBS) -lpython$(PYTHON_VERSION)
 libpyglib_gi_2_0_@PYTHON_BASENAME@_la_SOURCES = 	\
 	pyglib.c		\
 	pyglib.h		\
--- gi/_glib/Makefile.in.link	2011-08-22 05:36:17.000000000 +0200
+++ gi/_glib/Makefile.in	2011-08-22 05:36:29.000000000 +0200
@@ -299,7 +299,7 @@
 pyglib_LTLIBRARIES = _glib.la
 common_ldflags = -module -avoid-version $(am__append_1)
 libpyglib_gi_2_0_@PYTHON_BASENAME@_la_CFLAGS = $(GLIB_CFLAGS)
-libpyglib_gi_2_0_@PYTHON_BASENAME@_la_LIBADD = $(GLIB_LIBS) $(FFI_LIBS)
+libpyglib_gi_2_0_@PYTHON_BASENAME@_la_LIBADD = $(GLIB_LIBS) $(FFI_LIBS) -lpython$(PYTHON_VERSION)
 libpyglib_gi_2_0_@PYTHON_BASENAME@_la_SOURCES = \
 	pyglib.c		\
 	pyglib.h		\
--- gi/Makefile.am.link	2011-08-22 05:42:03.000000000 +0200
+++ gi/Makefile.am	2011-08-22 05:42:45.000000000 +0200
@@ -79,7 +79,7 @@
 	-export-symbols-regex "init_gi_cairo|PyInit__gi_cairo"
 _gi_cairo_la_LIBADD = \
 	$(GI_LIBS) \
-	$(PYCAIRO_LIBS)
+	$(PYCAIRO_LIBS) -lpython$(PYTHON_VERSION)
 _gi_cairo_la_SOURCES = pygi-foreign-cairo.c
 
 
--- gi/Makefile.in.link	2011-08-22 05:42:07.000000000 +0200
+++ gi/Makefile.in	2011-08-22 05:43:10.000000000 +0200
@@ -406,7 +406,7 @@
 
 _gi_cairo_la_LIBADD = \
 	$(GI_LIBS) \
-	$(PYCAIRO_LIBS)
+	$(PYCAIRO_LIBS) -lpython$(PYTHON_VERSION)
 
 _gi_cairo_la_SOURCES = pygi-foreign-cairo.c
 pygi_LTLIBRARIES = _gi.la $(am__append_1)
